### PR TITLE
fix(OverviewDetailTab): display mesos start and elected time

### PR DESCRIPTION
**Pull request by nLight**
Fixes never ending loaders for start and elected time.

screen shot 2018-04-17 at 11 53 40

Closes DCOS-22170

### CODE REVIEW

File: tests/pages/system/overview/ClusterDetails.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+describe("Cluster Details", function() {
```
Comment (juliangieseke) [inline tests/pages/system/overview/ClusterDetails.js]: there has to be a `-cy` suffix in the filename for the "runner" script.

Comment (juliangieseke) [reply tests/pages/system/overview/ClusterDetails.js]: just saw that I missed this on master 😢 these tests actually never run currently…
=> https://github.com/dcos/dcos-ui/pull/2904

Comment (nLight) [reply tests/pages/system/overview/ClusterDetails.js]: Thanks for catching it, fixed!

**Comment by DanielMSchmidt** [comment]
run integration tests

**Comment by DanielMSchmidt** [comment]
run integration tests

**Comment by Poltergeist** [comment]
Unfortunately, this is out of date with release/1.11 could you please rebase?

**Pull request merged**